themes/bootstrap3/templates/record/cover.phtml

```diff
@@ -1,8 +1,8 @@
 <?php /* Display thumbnail if appropriate: */ ?>
 <?php if ($cover): ?>
   <?php if ($this->link): ?><a href="<?=$this->escapeHtmlAttr($this->link)?>" data-lightbox-image><?php endif; ?>
-  <img alt="<?=$this->transEsc('Cover Image')?>" <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>class="recordcover" src="<?=$this->escapeHtmlAttr($cover); ?>"/>
+  <img alt="<?=$this->transEsc($this->link ? 'View Record Cover' : '') ?>" <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>class="recordcover" src="<?=$this->escapeHtmlAttr($cover); ?>"/>
```

Comment:
View Record Cover is not in the translation file, so we need to add that so it can go through the translation cycle.

Also, rather than having a conditional alt text based on link, can we instead put a title or aria-label on the `a` tag so that this is only read when there is a link, without having to mess with the `img` itself? Seems like that might be a slightly cleaner solution. (Maybe there's a good reason not to do it that way -- just a thought).

Reply:
I think ultimately we should add a way to customize this alt?

Reply:
Hi Chris, not sure whether costumization should really be necessary. If we provided a construct such as "Cover image for <book title> by <author(s)>" which might be a more "meaningful" than "Cover Image"  -- screen reader experience might suffer since title and author would be read (again) just seconds later. So a more explicit alt would mean more redundant information here. Or am I getting your idea of customizing this wrong?

Reply:
I guess the question is when we would want to set an alt here at all. I think as @ckaz says, most of the time, we don't need an alt at all because anything we could say would be useless or redundant. However, when the cover image has a link that is different from the title link, we probably need to express what it is doing. Most of the time, it's going to link to a larger version of the image, but there is a configurable feature that will sometimes link off to Google Books preview... perhaps we need to change the alt at the same time that we adjust the preview link.

Reply:
As I am poring over our accessibility test results from 2017, I came across the following recomendation (my translation): 

> The alt text "Cover Image" for the cover image is not really meaningful as it is not clear which it is the cover image for. Suggestion: Mention the work's title in the alt text or use aria-describedby to point to the ID of the respective title

Reply:
@ckaz, based on the new understanding we have after our WebAIM accessibility training, I think it is better to have alt="" than to describe cover images. They provide no useful information that is not already elsewhere on the page, so describing them just introduces annoying redundancies. When the images are linked to somewhere that is not linked from elsewhere on the page, describing them is more important, of course... but if the link only goes to another image, perhaps the whole thing should be aria-hidden, because cover images aren't going to be meaningful for screen reader users.

Reply:
Yes, you are right, so aria-hiding them all could be the option of choice.

Reply:
We had a course with the accessibility cracks here, they said that hiding "decorative" elements is perfectly fine and that we could also hide images by using emply alt tags